internal/data/metric.go

```diff
@@ -122,66 +237,171 @@ type internalMetric struct {
 	doubleDataPoints    []DoubleDataPoint
 	histogramDataPoints []HistogramDataPoint
 	summaryDataPoints   []SummaryDataPoint
```

Comment:
Looks like IntDataPoint slice is missing.

Reply:
you need to expand, github hides that :)

Reply:
Ah, OK :-)